views/components/content-box/macro.njk

```diff
@@ -1,6 +1,6 @@
-{% macro contentBox(id = false, isConstrained = true) %}
+{% macro contentBox(id = false, isConstrained = true, skipProse = false) %}
     <section class="content-box{% if isConstrained %} u-inner-wide-only{% endif %}"{% if id %} id="{{ id }}"{% endif %}>
-        <div class="s-prose u-constrained-content-wide">
+        <div class="{% if not skipProse %}s-prose {% endif %}u-constrained-content-wide">
```

Comment:
This was to allow us to put things in content boxes without inheriting all the `s-prose` styles (eg. all the list resets, which otherwise break this new component).

Reply:
Good call. I'd be just hard-coding the content box classes if I needed to skip this, but a flag makes sense 👍